config/default.yml

```diff
@@ -2799,6 +2799,12 @@ Style/HashSyntax:
   # Do not suggest { a?: 1 } over { :a? => 1 } in ruby19 style
   PreferHashRocketsForNonAlnumEndingSymbols: false
 
+Style/HashTransformMethods:
+  Description: 'Prefer `transform_keys` and `transform_values` over `each_with_object` and `map`.'
+  # Should only be enabled when Ruby 2.5 or greater is in use
+  Enabled: false
```

Comment:
This should be `pending`.

Reply:
When I set this to `pending`, I get a bunch of deprecation warnings from `bundle exec rake` even when I set `Enabled: true` in `.rubocop.yml`.

Reply:
This issue has been solved by #7664. Can you rebase with the latest master branch?